private/buf/cmd/buf/buf.go

```diff
@@ -150,23 +148,36 @@ func NewRootCommand(name string) *appcmd.Command {
 				Use:   "mod",
 				Short: "Configure and update buf modules.",
 				SubCommands: []*appcmd.Command{
-					modinit.NewCommand("init", builder, "", false),
+					modinit.NewCommand("init", builder, modInitDeprecationMessage, true), // TODO(alex): Think about this one (see TODO below).
+					export.NewCommand("export", builder, "", false),                      // TODO(alex): Move this command into the modexport package.
```

Comment:
The `export` command is agnostic to all inputs, not just modules. We should either:
1. Arbitrarily limit this to modules.
2. Keep it in the top-level namespace.

Reply:
I think that option 2 seems the most reasonable. I don't think it makes a lot of sense to go with 1, changing the nature of that functionality seems somewhat unreasonable and unwarranted, and if the entity that is being interacted with is not limited to modules, it should not be a `mod` level command.